litestar/config/app.py

```diff
@@ -131,6 +131,10 @@ class AppConfig:
         default_factory=list
     )
     """A list of callables returning async context managers, wrapping the lifespan of the ASGI application"""
+    server_lifespan: list[Callable[[Litestar], AbstractContextManager] | AbstractContextManager] = field(
+        default_factory=list
+    )
+    """A list of callables returning async context managers, wrapping the lifespan of the ASGI Server when called from the CLI."""
```

Comment:
Would a better name for this be service_cli_lifespan or something to match what the docstring states?